pubmed_dl/bin/pubmed-dl

```diff
@@ -23,7 +24,7 @@ settings = Settings()
 logging.basicConfig(level=settings.loglevel)
 
 
-def main(start_date: str, end_date: str, request_file: str):
+def main(start_date: str, end_date: str, request_file: str, progress_bar: Optional[bool] = typer.Argument(True)):
```

Comment:
I would make this a [typer option](https://typer.tiangolo.com/tutorial/options/help/) so that its invoked with something like `pubmed-dl ... --progress-bar`

Reply:
okay sure

Reply:
Hey, i added this functionality, so on default the progress bar shows up. If you don't want it then you can do `--no-progress-bar`